tempesta_fw/ss_skb.c

```diff
@@ -734,7 +744,8 @@ ss_skb_get_room(struct sk_buff *skb, char *pspt, unsigned int len, TfwStr *it)
 int
 ss_skb_cutoff_data(const TfwStr *hdr, int skip, int tail)
 {
-	int r;
+	int r, f_len;
+	void *f_ptr;
```

Comment:
I only see ```f_len``` usage in ```__skb_fragment()``` and ```BUG_ON()```. We don't care about BUG statements, because if you need more performance, then you can disable the checks. So we have only one usage of the variable, and it's better to avoid the variable.